### DISCUSSION TITLE
BigQuery integration / Automatic data export to bucket

### DISCUSSION THREAD

**Discussion by daallgeier** [plausible/analytics#1777] [Anything Else]
To enable data analytics over big data sets, multiple web properties, and easy reporting a connection to BigQuery is super helpful.  

Google Analytics supports this through Analytics 360.  

I do not expect you to directly integrate it at this point but offering a regular (daily) csv export to a Google Cloud Bucket (or S3, I guess 😉) would enable this as well. Implementing it this way will also enable other customers to use these exports in more ways than one.   Once the data is in that bucket I can query the bucket and push it into BigQuery automatically.  Other tools can also do similar things with new data in a bucket.  

Obviously the implementing a BigQuery connection directly would be preferred for me but I am happy with the csv to bucket path.

**Comment by metmarkosaric** [comment]
This might be useful to you: There's now a CloudQuery plugin for Plausible. It extracts information from Plausible stats API and loads it into any supported CloudQuery destination such as Snowflake, BigQuery, S3 etc. See https://www.cloudquery.io/docs/plugins/sources/plausible/overview

**Reply by motiejunas** [reply]
@metmarkosaric there is no plugin like that

**Reply by cnkk** [reply]
@motiejunas looks like the plugin got deprecated: https://community.cloudquery.io/t/cloudquery-integrations-deprecation-notice/972/7
But there is a link to the binary of the plugin in the linked thread.
